extensions/podman/packages/extension/src/extension.ts

```
@@ -1292,8 +1296,14 @@
~
~  // bind classes to the Inversify container
   inversifyContainer.bind(PodmanInstall).toSelf().inSingletonScope();
   const podmanInstall = inversifyContainer.get(PodmanInstall);
 
+  if (telemetryLogger) {
+    await initializeCertificateDetection(telemetryLogger);
+  }
+
+  const podmanInstall = new PodmanInstall(extensionContext, telemetryLogger);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'podmanInstall'. Consider to delete it or rename it.

'podmanInstall' is defined here:


(lint/suspicious/noRedeclare)